parsil/src/validate.rs

```diff
@@ -390,14 +389,6 @@ impl<'a, C: ContextProvider> AstVisitor for SqlValidator<'a, C> {
             q.order_by.is_none(),
             ValidationError::UnsupportedFeature("ORDER BY".into())
         );
-        if let Some(l) = &q.limit {
-            let limit_value = const_eval(l).map_err(|_| ValidationError::InvalidLimitExpression)?;
-            let max_limit = U256::from(C::MAX_NUM_OUTPUTS);
-            ensure!(
-                limit_value <= max_limit,
-                ValidationError::LimitTooHigh(C::MAX_NUM_OUTPUTS)
-            );
-        }
```

Comment:
@delehef why removing this validation check? I think we should ensure that the limit specified by the user is less than the upper bound we support in circuit, no?

Reply:
You're right, but I'd prefer to do it at the instantiation of the `ParsilSettings`.

Reply:
Sounds good to me, not sure how to do it properly now since the `ParsilSettingsBuilder` doesn't return an error now, but I guess you already know how to check it ;)

Reply:
It will soon :)